### PR TITLE
Export as transparent PNG

### PR DESCRIPTION
**Pull request by terryyoon-pdftron**
https://linear.app/pdftron/issue/CPM-5/[support-30542]-nitrolabs-export-signature-to-png-transparent

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -212,6 +212,7 @@ pageNumber | int | the page to be converted; if the value does not refer to a va
 dpi | double | the output image resolution
 exportFormat | string | one of [`Config.ExportFormat`](./src/Config/Config.ts) constants
 filePath | string | local file path to pdf
+transparent | boolean | (only relevant when exported as PNG) whether the background of the image is transparent or opaque white
```
Comment (JamesTron) [inline API.md:215]:
```suggestion
transparent | boolean | (only relevant when exported as PNG) whether the background of the image is transparent or the color of the PDF page (typically white)
```

**Review by JamesTron — CHANGES_REQUESTED** [review]
Looks good, minor detail in documentation, no need to re-review after.

File: src/DocumentView/DocumentView.tsx [hunk 2]
```diff
@@ -1042,10 +1042,10 @@ export class DocumentView extends PureComponent<DocumentViewProps, any> {
     return Promise.resolve();
   }
 
-  exportAsImage = (pageNumber: number, dpi: number, exportFormat: Config.ExportFormat): Promise<void | string> => {
+  exportAsImage = (pageNumber: number, dpi: number, exportFormat: Config.ExportFormat, transparent: boolean): Promise<void | string> => {
```
Comment (sgong-pdftron) [inline src/DocumentView/DocumentView.tsx:1045]: this variable needs to be optional otherwise it'd be breaking API

File: index.ts [hunk 3]
```diff
@@ -12,10 +12,10 @@ interface RNPdftron {
     getSystemFontList() : Promise<string>;
     clearRubberStampCache() : Promise<void>;
     encryptDocument(filePath: string, password: string, currentPassword: string) : Promise<void>;
-    pdfFromOffice(docxPath: string, options?: {applyPageBreaksToSheet?: boolean, displayChangeTracking?: boolean, excelDefaultCellBorderWidth?: number, 
+    pdfFromOffice(docxPath: string, options?: {applyPageBreaksToSheet?: boolean, displayChangeTracking?: boolean, excelDefaultCellBorderWidth?: number,
             excelMaxAllowedCellCount?: number, locale?: string}) : Promise<string>;
     pdfFromOfficeTemplate(docxPath: string, json: object) : Promise<string>;
-    exportAsImage(pageNumber: number, dpi: number, exportFormat: Config.ExportFormat, filePath: string) : Promise<string>;
+    exportAsImage(pageNumber: number, dpi: number, exportFormat: Config.ExportFormat, filePath: string, transparent: boolean) : Promise<string>;
```
Comment (sgong-pdftron) [inline index.ts:18]: same here, needs to be optional

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

**Review by sgong-pdftron — APPROVED** [review]

**Review by JamesTron — APPROVED** [review]

**Pull request merged**